frontend/src/components/Order/components/ContactDetails/ContactDetails.jsx

```diff
@@ -9,6 +9,8 @@ const ContactDetails = ({ contactData, setDataOnSubmit }) => {
     contactData?.recipient ? contactData?.recipient === 'anotherPerson' : false,
   );
 
+  const userData = JSON.parse(localStorage.getItem('userfetchedData'));
```

Comment:
Consider the case where such information is not available in localStorage. JSON.parse may throw an error